lib/adapters/leveldb/index.js

```diff
@@ -10,7 +10,16 @@ function requireLeveldown() {
   // de-optimalization
   try {
     originalLeveldown = require('leveldown');
-  } catch (e) {}
+  } catch (err) {
+      if (err && err.message && err.message.match('Module version mismatch')) {
+          throw new Error([
+            err.message,
+            'This generally implies that leveldown was built with a different',
+            'version of node than that which is running now.  You may try',
+            'fully removing and reinstalling PouchDB or leveldown to resolve.'
+          ].join(' '));
+      }
```

Comment:
I *love* this new error message and think it's much more helpful, but if we `throw` within a `catch`, then that removes the whole value of the `try/catch`. The point here was that, if you were on a system that didn't support leveldown, you could use some other *down (sqldown, memdown, etc.).

I'd suggest:

```js
if (typeof console !== 'undefined' && typeof console.warn === 'function') {
  console.warn(errorMessage);
}
```